### ISSUE TITLE
Ignored File Warnings

### ISSUE THREAD

**Issue by kentcdodds** [MoOx/eslint-loader#44]
I had [this problem](https://gitter.im/eslint/eslint?at=5576019cf40a067d1c9bd7a8)

Summarized:

> Getting this:
> 
> ```
> WARNING in ./app/components/dev/custom.ignored.js
> 
> /Users/kentcdodds/Developer/alianza/atac5/app/components/dev/custom.ignored.js
>   0:0  warning  File ignored because of your .eslintignore file. Use --no-ignore to override
> 
> ✖ 1 problem (0 errors, 1 warning)
> ```
> 
> Any way I can silence that particular warning?
> I'm explicitely ignoring that file, so I don't need/want a warning about it...
> Just started getting this when I upgraded from 0.19.0 to the latest

Then later, someone mentioned [the docs](http://eslint.org/docs/user-guide/configuring#ignored-file-warnings) on the subject saying:

> Documentation is stating that you shouldn't see this warning when you run eslint on a directory or multiple files, only when you run it on a single file that is being ignored
> Is that not what you are seeing?

And, I explained that I'm using the `eslint-loader`, to which he responded:

> All of the build systems usually do glob patterns first and then run eslint for each file separately
> Grunt and Gulp do the same thing

Is there any way that we could get around this without having to `quiet` all of our warnings?

**Comment by MoOx** [comment]
I guess the only is we have is to manually ignore this warning.
Maybe the one who add this PR have an idea about that ? /cc @mgtitimoli

**Comment by MoOx** [comment]
Since it make no sense to send a warning for a file that should be ignored, I just ignored the ignore warning :)

**Comment by MoOx** [comment]
"Fix" released as 0.13

**Comment by kentcdodds** [comment]
> I just ignored the ignore warning :)

nice. Thanks @MoOx!
